core/src/main/resources/error/error-classes.json

```diff
@@ -630,6 +630,11 @@
       "Input schema <jsonSchema> can only contain STRING as a key type for a MAP."
     ]
   },
+  "INVALID_LIKE_PATTERN" : {
+    "message" : [
+      "The pattern '<pattern>' is invalid, <message>."
```

Comment:
It would be nice to avoid the `message` but it is useful in this case. @cloud-fan @srielau WDYT?

Reply:
Is it always the same message? If so we could map it.

Reply:
Actually, the message comes from Spark's code, see:
https://github.com/apache/spark/blob/70ec696bce7012b25ed6d8acec5e2f3b3e127f11/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/StringUtils.scala#L50
and there are just 2 messages:
1. https://github.com/apache/spark/blob/70ec696bce7012b25ed6d8acec5e2f3b3e127f11/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/StringUtils.scala#L59
2. https://github.com/apache/spark/blob/70ec696bce7012b25ed6d8acec5e2f3b3e127f11/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/util/StringUtils.scala#L61

@panbingkun Could you place the messages to the JSON files instead of leaving them in the source code, please.

Reply:
Done